### DISCUSSION TITLE
minio 的url能否指定多个ip？

### DISCUSSION THREAD

**Discussion by feehoo** [juicedata/juicefs#2822] [General]
minio 的url能否指定多个ip？minio是多机集群，url如果只支持单个ip的话，可能存在单点故障。

**Comment by zhijian-pro** [comment]
不支持。minio 多机多 driver 模式下官方推荐使用负载均衡器 NGINX 或者 HAProxy 配合使用。这样可以暴露同一入口的同时一定程度保证高可用。你可以看这个[文档](https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html?ref=docs-redirect)
